src/main/java/core/basesyntax/serviceimpl/FruitTransactionValidatorImpl.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionValidator;
+
+public class FruitTransactionValidatorImpl implements FruitTransactionValidator<FruitTransaction,
+        FruitTransaction.Operation> {
```

Comment:
The line break in the middle of the class declaration is unconventional and can be confusing. Consider keeping the class declaration on a single line for better readability.